packages/block-editor/src/components/url-input/index.js

```diff
@@ -207,6 +212,10 @@ class URLInput extends Component {
 				break;
 			}
 		}
+
+		if ( isFunction( this.props.onKeyDown ) ) {
```

Comment:
Nitpick, but I think it's better to do `if ( this.props.onKeyDown ) {`, as that'll throw an error if the implementor does something wrong, providing a better feedback loop.

Reply:
Fixed.